web/app/integration/specs/daemonset/daemonset-table.js

```diff
@@ -0,0 +1,16 @@
+const assert = require('assert');
+    describe('table check', function() {
+    it('should check the page for proper table structure', () => {
+        browser.url('http://127.0.0.1:7777/daemonsets');
+        browser.waitUntil(() => {return $('table').isExisting();
+          }, 10000, 'expected table to exist');
+        http_headers=["Namespace", "Daemon Set","Meshed", "Success Rate", "RPS", "P50 Latency", 
+        "P95 Latency", "P99 Latency", "Grafana"];
+        tcp_headers=["Namespace","Daemon Set","Meshed","Connections", "Read Bytes / sec","Write Bytes / sec", 
+"Grafana"];
+        const http=$$("table")[0].$("thead").$$("th").map(item=>item.getText());
+        assert(http.join('')==http_headers.join(''));
+        const tcp=$$("table")[1].$("thead").$$("th").map(item=>item.getText());
+        assert(tcp.join('')==tcp_headers.join(''));
```

Comment:
in JS the preferred comparison operator is `===` as opposed to `==`. Check out [this](https://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons) stackoverflow post on the reason's why.

Reply:
Oh my! JS sure has it's quirks. Will change tests to the appropriate equality symbol. Thank you for pointing this out!